server/src/main/scala/actors/daily/TerminalDayFlightActor.scala

```diff
@@ -78,11 +81,26 @@ class TerminalDayFlightActor(
     restorer.finish()
   }
 
+  private def matchesScheduledAndVoyageNumber(fws: ApiFlightWithSplits, scheduled: SDateLike, voyageNumber: VoyageNumberLike) = {
+    fws.apiFlight.Scheduled == scheduled.millisSinceEpoch && fws.apiFlight.VoyageNumber.numeric == voyageNumber.numeric
+  }
+
+  private def RedListCountDiffWith(counts: Iterable[RedListPassengers]): FlightsWithSplitsDiff = {
```

Comment:
Shall we lower-case the first letter of this function to follow function naming conventions?